src/create_venv.sh

```diff
@@ -1,136 +1,206 @@
 #!/usr/bin/env bash
 
+# Copyright (c) YugaByte, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied.  See the License for the specific language governing permissions and limitations
+# under the License.
+#
+
+[[ "${_YB_CREATE_VENV_SHLIB:=""}" == "yes" ]] && return 0
+_YB_CREATE_VENV_SHLIB=yes
+VERBOSE=${VERBOSE:-false}
```

Comment:
VERBOSE can clash with a variable in the user script. Can we use _YB_VERBOSE or some consistent notation like that?